src/pymor/vectorarrays/list.py

```diff
@@ -743,7 +743,7 @@ def axpy(self, alpha, x):
         super().axpy(alpha, x)
 
     def __str__(self):
-        return f'ListVectorArrayView of {len(self._list)} {str(self.vector_type)}s of dimension {self.dim}'
+        return f'ListVectorArrayView of {len(self._list)} of dimension {self.dim}'
```

Comment:
```suggestion
        return f'ListVectorArrayView of {len(self._list)} vectors of dimension {self.dim}'
```
(Should also be applied to `ListVectorArray.__str__`)

Reply:
Done.
What do you think about `ListVectorArray.__str__` using `self.space` and `ListVectorArrayView.__str__` using `self.dim`?

Reply:
Why should a view have a different string representation? I would expect the same, maybe with a hint that the view is a view.

Reply:
I agree. That was the state before this PR. Do you have a preference for which to use?

Reply:
I think I would prefer the space.